src/Stratis.Bitcoin.Features.Consensus/Rules/ConsensusRules.cs

```diff
@@ -134,4 +135,43 @@ public void SetErrorHandler(ConsensusRule errorHandler)
             }
         }
     }
+
+    /// <summary>
+    /// Consensus rules that represent a Proof-Of-Work blockchain.
+    /// </summary>
+    public class PowConsensusRules : ConsensusRules
```

Comment:
@dangershony can we maybe put these into seperate files? I believe there was some agreement made a while back on this ;)

Reply:
yeah I will do it on my next PR